src/client/gui/lib/catalogue/launch_form.dart

```diff
@@ -317,7 +322,19 @@ class _LaunchFormState extends ConsumerState<LaunchForm> {
     }
 
     initiateLaunchFlow(ref, launchRequest, mountRequests);
-    Scaffold.of(context).closeEndDrawer();
+
+    if (!configureNext) {
+      formState.reset();
+      mountFormState?.reset();
+      setState(() {
+        mountRequests.clear();
+        addingMount = false;
+      });
```

Comment:
Do you really need to clear those? Won't they just go away with the `Scaffold.of(context).closeEndDrawer();`?